### ISSUE TITLE
ECS Prerelease 2024-10-03: Oversized station scaling

### ISSUE THREAD

**Issue by Xansta** [daid/EmptyEpsilon#2185]
When I run the Close the Gaps scenario, I noticed that the huge stations show up as _really_ huge:
Undocked:
![Screenshot 2024-10-03 at 9 41 55 AM](https://github.com/user-attachments/assets/a8777d40-35eb-4981-a393-8121d64b3453)
Docked:
![Screenshot 2024-10-03 at 9 42 43 AM](https://github.com/user-attachments/assets/db490b2c-802f-4b2f-b628-c5145a66c54d)
For reference, here's what it looks like in the current stable release (2024-08-09):
Undocked:
![Screenshot 2024-10-03 at 9 44 28 AM](https://github.com/user-attachments/assets/f62cd52f-4d01-4195-a189-b418eb5fdb80)
Docked:
![Screenshot 2024-10-03 at 9 45 20 AM](https://github.com/user-attachments/assets/880163e6-cbc4-4a67-9053-e5450d185c5f)

This may be how it was always supposed to be. If so, the station shields are shown as far enough away from the station that they should cover the docked ship. I tested this out and it's not true - the enemy shots came right "through" the station shields to hit the player ship. The other station representations (small, medium, large) also seem a bit larger than usual. The huge station just magnifies this behavior.

**Comment by Xansta** [comment]
It seems to be affecting some ships, too. Here is a _very_ large freighter:
![Screenshot 2024-10-03 at 3 48 44 PM](https://github.com/user-attachments/assets/3f605305-52c6-4b05-a0db-bc04a4938ad1)

**Comment by daid** [comment]
Changes to render large ships as larger so their icon reflects the actual size better is intentional. Having the shields graphics extend beyond that wasn't intentional and seems to be the main issue.

**Comment by Xansta** [comment]
Makes sense to me that you're trying for a better visual scale. If the other things on the radar are to be scaled, then the player ship should probably also be scaled. The player ship compared to the freighter (just the icon, not the shields) makes the freighter look so much larger than the player ship. I can't tell if this is a scale issue for the freighter or the the player ship or both or neither (player ship is really that much smaller than the freighter). I also can't tell if this is the result of the scaling for the short range radar (helm in this case) or if the scaling will have similar issues in the long range radar (Science and maybe Relay). Since the long range radar range and the short range radar range can both be customized to something other than the defaults of 30k and 5k respectively, I imagine that whatever scaling algorithm you use has to work for both of those consoles. For Relay or strategic map, you might have different scaling parameters apply.

**Comment by daid** [comment]
player ships are scaled in exactly the same way. But, for long rectangular ships, the length size is what is scaling the icon, so that might make it look "off" for the freighters...

**Comment by daid** [comment]
I think the situation will be a lot better now. But might need more tweaks in the future, we'll see.
